app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/test/chapter2/customfailurehandler/CustomFailureHandlerTest.java

```diff
@@ -0,0 +1,40 @@
+package com.example.android.architecture.blueprints.todoapp.test.chapter2.customfailurehandler;
+
+import android.graphics.Color;
+
+import com.example.android.architecture.blueprints.todoapp.R;
+import com.example.android.architecture.blueprints.todoapp.test.BaseTest;
+import com.example.android.architecture.blueprints.todoapp.test.chapter2.custommatchers.CustomViewMatchers;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import static android.support.test.espresso.Espresso.onView;
+import static android.support.test.espresso.action.ViewActions.click;
+import static android.support.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static android.support.test.espresso.action.ViewActions.typeText;
+import static android.support.test.espresso.assertion.ViewAssertions.matches;
+import static android.support.test.espresso.matcher.ViewMatchers.hasErrorText;
+import static android.support.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static android.support.test.espresso.matcher.ViewMatchers.withId;
+import static android.support.test.espresso.matcher.ViewMatchers.withText;
+
+public class CustomFailureHandlerTest extends BaseTest{
```

Comment:
There should be a space after `BaseTest`and before a bracket:
`extends BaseTest {`